.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: go
+
+go:
+  - 1.7.x
+  - 1.8.x
+  - 1.9.x
+  - master
+
+matrix:
+ fast_finish: true
+ allow_failures:
+   - go: master
+
+sudo: true
+
+before_install:
+  - sudo apt-get install pkg-config libzmq3-dev build-essential python3-pip
+  - pip3 install --upgrade pip
+
+install:
+  - go get golang.org/x/tools/cmd/goimports
+
+script:
+  - go get -tags zmq_3_x github.com/gopherdata/gophernotes
```

Comment:
The new version supports zeromq 4.x.x. Thus, I don't think you will need the tag here (because we are using a different zmq package) and the `libzmq3-dev` install might need to be modified.

Reply:
the thing is that the latest docker image on which travis-ci runs is "trusty".
installing `zmq4` there must be done via either `ppa` archives or from sources.

Reply:
Hmmm... I see.  Well, I think we should have zeromq 4.x.x.  Is the problem with getting the tar ball and installing just that it will take time in the build?

Reply:
yep.

Reply:
we could start with `v3` and then migrate to building `v4`, like is done here:
https://github.com/zeromq/gomq/blob/master/.travis.yml
(but requesting an explicit tag/branch instead of building `master`)

Reply:
That might work out well. I mean, having a slow build is probably not the end of the world here, so if we want to just get the tar ball and install we could. Can you specify custom images similar to with circle ci?  Just wondering, because we could create a public build image with zmq 4.x.x installed.

Reply:
perhaps. I vaguely remember somebody doing something perhaps related to that for Gonum (and the OpenBlas build)

Reply:
(let me check that. well, after my last lectures for next week are ready. or ready-ish)